app/form_models/publishers/job_application/tag_form.rb

```diff
@@ -1,8 +1,30 @@
 class Publishers::JobApplication::TagForm
   include ActiveModel::Model
   include ActiveModel::Validations
+  include ActiveModel::Attributes
 
-  attr_accessor :job_applications
+  attribute :status, :string
+  attr_accessor :job_applications, :origin
 
   validates_length_of :job_applications, minimum: 1
+  validates_presence_of :status, on: :update_tag
+
+  def job_application_ids
+    return [] if job_applications.blank?
+
+    job_applications.pluck(:id)
+  end
+
+  def update_job_application_statuses
```

Comment:
Think this needs a comment or a better name - I can't understand what this method is doing, or why it is here